commands/deployments/dep_undeploy.go

```diff
@@ -74,6 +81,7 @@ func init() {
 
 	DeploymentsCmd.AddCommand(undeployCmd)
 	undeployCmd.PersistentFlags().BoolVarP(&purge, "purge", "p", false, "To use if you want to purge instead of undeploy")
+	undeployCmd.PersistentFlags().BoolVarP(&stopOnError, "stop-on-error", "", false, "By default if an error occurs during the undeployment, the error is bypassed and the un-deployment continues. This flag allows to stop if an error occurs.")
```

Comment:
Would need to document this new option `stop-on-error` in doc/cli.rst section `Undeploy a deployment`

Reply:
Yes

Reply:
Done